solutions/swb-reference/integration-tests/support/complex/accountHelper.ts

```diff
@@ -0,0 +1,153 @@
+import { PolicyDocument } from '@aws-cdk/aws-iam';
+import { GetBucketPolicyCommandOutput, PutBucketPolicyCommandInput } from '@aws-sdk/client-s3-control';
+import { AwsService } from '@aws/workbench-core-base';
+import { IamHelper } from '@aws/workbench-core-datasets';
+import _ from 'lodash';
+import Setup from '../setup';
+import Settings from '../utils/settings';
+
+export class AccountHelper {
+  private _awsSdk: AwsService;
+  private _settings: Settings;
+  public constructor(awsSdkClient: AwsService) {
+    const setup: Setup = new Setup();
+    const settings: Settings = setup.getSettings();
+    this._settings = settings;
+    this._awsSdk = awsSdkClient;
+  }
+
+  public async listOnboardedAccounts(): Promise<{[id: string]: string}[]> {
+    const queryParams = {
+      index: 'getResourceByCreatedAt',
+      key: { name: 'resourceType', value: 'account' }
+    };
+    const response = await this._awsSdk.helpers.ddb.query(queryParams).execute();
+    let accounts: {[id: string]: string}[] = [];
+    if (response && response.Items) {
+      accounts = response.Items.map((item) => {
+        return item as unknown as {[id: string]: string};
+      });
+    }
+    return accounts;
+  }
+
+  public async deleteDdbRecords(accountId: string, awsAccountId: string): Promise<void> {
+    await this._awsSdk.helpers.ddb
+      .delete({ pk: `ACC#${accountId}`, sk: `ACC#${accountId}` }).execute();
+    await this._awsSdk.helpers.ddb
+      .delete({ pk: `AWSACC#${awsAccountId}`, sk: `ACC#${accountId}` }).execute();
+  }
+
+  public async verifyBusPermitsAccount(awsAccountId: string): Promise<boolean> {
```

Comment:
Should `permits` be `permission`?

Typo with the `t`?

Reply:
It's not a typo, "permits" is used as a verb. This method verifies if the event bus _permits_ the hosting account to access it.